_posts/2016-01-30-iOS-Junior-Interviews.markdown

```diff
@@ -0,0 +1,300 @@
+
+- [StackOverflow jobs](http://stackoverflow.com/jobs)
+- [Natasha The Robot's Swift Jobs](https://www.natashatherobot.com/swift-jobs/)
+- [Core Intuition's Jobs Board](http://jobs.coreint.org)
+- [Apple's Jobs Site](https://www.apple.com/jobs/us/)
+
+These kind of things work, I got my first programming job through [CocoaDev Jobs](http://www.cocoabuilder.com/archive/cocoa/174142-ann-cocoadev-jobs-board.html).
+
+These are great, but in my opinion the best way to find out what companies are hiring is to attend meet-ups in the city. In NYC we have 3-4 big iOS meet-ups a month, and all of them give a chance for people looking for others to work to speak up. You might not live in as big of a city, but I used to travel an hour to Manchester to attend a meet-up, they would have the same thing, I almost became a full-time rubyist because of it. I have quite a few friends in NYC who started their careers by talking to someone at a meet-up.
+
+Generic job websites can be good, for example Artsy's job postings are on [AngelList](https://angel.co/artsy) and [Glassdoor](https://www.glassdoor.co.uk/Job/Artsy-Jobs-E793485.htm) but the vast majority of the incoming applicants from them feel like they were sent to a lot of companies. They are given a low priority when compared to direct inquiries because of this. I can't speak from experience but [Ladders](https://www.theladders.com), [LinkedIn](https://www.linkedin.com), [Hired](http://hired.com) and [weworkremotely](https://weworkremotely.com) are probably worth a look too.
+
+If you see an advert for a mobile developer, but they don't mention juniors, you should apply anyway. We've never had junior positions on our jobs pages, but have hired some who inquired anyway. Don't pretend to be more senior than you are though, set the right expectations. 
+
+Finally, consult your network, talk to people at meet ups, sneakily contribute to open source projects with people you want to talk to and then email the project owner after they've got to know your name.
+
+### On a Coffee
+
+Know what goes a long way? Talking with someone before applying. I regularly get coffee with applicants or potential applicants when I'm in NYC. Coffee can be hard when applying for junior roles when remote, but so far, to my knowledge I've not talked to anyone who hires a junior as a remote developer. I'm not sure it could work unless the entire team was remote, or the junior developer had a lot of experience and was only a junior in the sense that they were just hitting the work-force.
+
+These are 15-30 minute informal chats, but from an employers perspective they are definitely a good way to filter candidates. A coffee is time-cheap, interviews are time-expensive. So ask someone on the dev team to do them, you'll usually get a yes. Tamar Nachmany has some [great, salient points](https://booleancubes.tumblr.com/post/137057622039/the-networking-with-awesome-strangers-checklist) on the right way to pitch these emails.
```

Comment:
Also might be good to note that being super prepared for coffee can be helpful - if you show that you've done your homework, you can turn coffee into an interview.